src/OpenLoco/src/Map/MapGenerator.cpp

```diff
@@ -215,8 +215,7 @@ namespace OpenLoco::World::MapGenerator
             const bool isValidMountainIndex = generateMountains && randomHillIndex >= hillShapesObj->hillHeightMapCount;
             if (!(isValidHillIndex || isValidMountainIndex))
             {
-                printf("Bail: flags = %d, randomHillIndex = %d, hillHeightMapCount = %d, mountainHeightMapCount = %d\n",
-                    enumValue(topographyFlags), randomHillIndex, hillShapesObj->hillHeightMapCount, hillShapesObj->mountainHeightMapCount);
+                printf("Bail: flags = %d, randomHillIndex = %d, hillHeightMapCount = %d, mountainHeightMapCount = %d\n", enumValue(topographyFlags), randomHillIndex, hillShapesObj->hillHeightMapCount, hillShapesObj->mountainHeightMapCount);
```

Comment:
Use Logging, should find a way to block printf from compiling 😄

Reply:
Haha, printf is very ingrained in my muscle memory, I suppose. But tbh, I didn't even mean for this to make it to the end of the PR.

Reply:
I actually think we don't log not enough but I see, if you decide to leave it in, Logging::verbose and it will hardly hurt the performance as it bails if it doesn't pass the level filter.